wafer/settings.py

```diff
@@ -202,7 +202,7 @@
 REGISTRATION_FORM = 'wafer.registration.forms.WaferRegistrationForm'
 
 # Forms:
-CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap5"
+CRISPY_ALLOWED_TEMPLATE_PACKS = ["bootstrap5", ]
```

Comment:
```suggestion
CRISPY_ALLOWED_TEMPLATE_PACKS = ["bootstrap5"]
```
We don't need the trailing comma for lists.